### DISCUSSION TITLE
Firebase login failed in the home page

### DISCUSSION THREAD

**Discussion by fimoefive** [nss-evening-cohort-14/evening-client-side#177] [Help Me!] [answered]
my google login failed when signing in from the home page. It says failed with a status of 400?  
Here is my console error: 
auth.js?2ce1:67 Uncaught 
t {code: "auth/internal-error", message: "{"error":{"code":400,"message":"API key not valid.…son":"badRequest"}],"status":"INVALID_ARGUMENT"}}", a: null}
a: null
code: "auth/internal-error"
message: "{"error":{"code":400,"message":"API key not valid. Please pass a valid API key.","errors":[{"message":"API key not valid. Please pass a valid API key.","domain":"global","reason":"badRequest"}],"status":"INVALID_ARGUMENT"}}"
__proto__: Error

I made sure that I fully debug your code and I'm having no other errors. It says a "badRequest" I double checked my firebaseKeys in the .env file and everything matched. Also I checked my Api for Boards and pins in Postman and everything ran and displayed well.

**Comment by drteresavasquez** [comment, marked as answer]
1. Check your .env file to make sure that you have the right vars
2. Make sure you have authentication set up on your project on Firebase

**Reply by drteresavasquez** [reply]
Also: https://github.com/nss-evening-cohort-14/evening-client-side/discussions/125

**Reply by jrobinson0529** [reply]
I don't know if this is mentioned in the videos, I missed it if it was but you need to go to
- Firebase console  > Authentication > Click the "Sign-in method" tab > Scroll down below "Sign-in providers" > click the "Add domain" button and add your deployed site domain in there.

**Reply by drteresavasquez** [reply]
@jrobinson0529 good call! It is mentioned there and on the Netlify deploy docs as well. I'll make sure to highlight that.

**Reply by drteresavasquez** [reply]
https://github.com/nss-evening-cohort-14/webpack-template#deploying-on-netlify

**Comment by drteresavasquez** [comment]
@fimoefive are you still needing help with this? Have not seen a response to the other suggestions given.

**Reply by fimoefive** [reply]
I had to restart VSC by "control, c" then relaunching rpm i and npm start after I setup my user/owner Authentication.
Then my Pinterest app was up and running.
Thank you for your help! : )

➜  ASSIGNMENT-Firebase-Pinterest git:(mjs-render) npm i
npm WARN PINTEREST@1.0.0 No repository field.

audited 1200 packages in 4.77s

72 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
   │                                                               │
   │      New major version of npm available! 6.14.11 → 7.6.2      │
   │   Changelog: https://github.com/npm/cli/releases/tag/v7.6.2   │
   │               Run npm install -g npm to update!
